scripts/clear-counters.py

```diff
@@ -4,87 +4,106 @@
 This script clear counters on devices
 """
 
-# disabling duplicate-code for scripts as this is expected between scripts
-# pylint: disable=R0801
-
+import asyncio
 import logging
-import ssl
+import traceback
 from argparse import ArgumentParser
 from getpass import getpass
 
-from jsonrpclib import jsonrpc
+from rich.logging import RichHandler
 
 from anta.inventory import AntaInventory
 
-# pylint: disable=protected-access
-ssl._create_default_https_context = ssl._create_unverified_context
+logger = logging.getLogger(__name__)
+
+
+def setup_logging(level: str = "info") -> None:
+    """
+    Configure logging for check-devices execution
+
+    Helpers to set logging for
+    * anta.inventory
+    * anta.result_manager
+    * check-devices
+
+    Args:
+        level (str, optional): level name to configure. Defaults to 'critical'.
+    """
+    loglevel = getattr(logging, level.upper())
+
+    # FORMAT = "%(asctime)s:%(levelname)s:%(message)s"
+    FORMAT = "%(message)s"
+    logging.basicConfig(
+        level=loglevel, format=FORMAT, datefmt="[%X]", handlers=[RichHandler()]
+    )
+    logging.getLogger("anta.inventory").setLevel(loglevel)
+    logger.setLevel(loglevel)
 
 
-def clear_counters(inventory: AntaInventory, enable_pass: str) -> None:
+async def clear_counters(inv: AntaInventory, enable_pass: str) -> None:
     """
     clear counters
     """
-    devices = inventory.get_inventory(established_only=True)
+    await inv.connect_inventory()
+    devices = inv.get_inventory(established_only=True)
     for device in devices:
-        switch = device.session  # type: ignore
-        host = str(device.host)  # type: ignore
         try:
-            if device.hw_model in ["cEOSLab", "vEOS-lab"]:  # type: ignore
-                switch.runCmds(
-                    1, [{"cmd": "enable", "input": enable_pass}, "clear counters"]
+            if device.hw_model in ["cEOSLab", "vEOS-lab"]:
```

Comment:
I would rewrite this to build first the line of commands based on the device hw_model and then call only once await + logger